### PR TITLE
dolt merge --no-ff

### PR DESCRIPTION
**Pull request by bheni**

### CODE REVIEW

File: go/cmd/dolt/commands/merge.go [hunk 1]
```diff
@@ -307,6 +362,12 @@ func executeMerge(ctx context.Context, squash bool, dEnv *env.DoltEnv, cm1, cm2
 		}
 	}
 
+	return commitMergedRoot(ctx, squash, dEnv, mergedRoot, workingDiffs, cm2, tblToStats)
+}
+
+func commitMergedRoot(ctx context.Context, squash bool, dEnv *env.DoltEnv, mergedRoot *doltdb.RootValue, workingDiffs map[string]hash.Hash, cm2 *doltdb.Commit, tblToStats map[string]*merge.MergeStats) errhand.VerboseError {
```
Comment (andrew-wm-arthur) [inline go/cmd/dolt/commands/merge.go:368]: does this actually commit anything? I think it needs a different name

**Review by andrew-wm-arthur — APPROVED** [review]
LGTM, one comment

**Comment by timsehn** [comment]
BATS looks good.

**Comment by timsehn** [comment]
We should get this in for the release.

**Pull request merged**